python/oneflow/test/tensor/test_tensor_indexing2.py

```diff
@@ -903,6 +905,11 @@ def _test_ellipsis_tensor(test_case, device):
     test_case.assertEqual(x[..., idx].tolist(), [[0, 2], [3, 5], [6, 8]])
     test_case.assertEqual(x[idx, ...].tolist(), [[0, 1, 2], [6, 7, 8]])
 
+    # test getitem:scalar indexed with scalar
+    y = flow.randn((), device=device)
```

Comment:
```suggestion
    y = flow.tensor(1.0).to(device)
```

这样写吧，`flow.randn(())` 不容易一眼看出来这是个 scalar tensor